### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>libraries\IRremote ne fonctionne pas avec attiny1616

### DISCUSSION THREAD

**Discussion by gparoz** [community/community#121000] [Discussions]
### Select Topic Area

Bug

### Body

arduino ide 2.3.2 libraries\IRremote\src\TinyIRReceiver.hpp 4.3.1
programme:   exemple----> TinyReceiver :
fonctionne parfaitement avec NANO sur toutes les broches
Ne fonctionne pas avec mega tinycore et attiny1616, sur aucune pin !
 la bibliothèque supporte attiny1616 dans GitHub,
 le port choisi,  A1 (14)  fonctionne avec  
Serial.println(F("START " __FILE__ " from " __DATE__ "\r\nUsing library version " VERSION_TINYIR));
et retourne bien sur le moniteur serial         ---->  attiny1616 et pin 14 
ainsi que    #define TRACE    
 qui envoie bien une longue  trame sur Serial.print   à chaque pression d'une touche 
ce qui confirme la fonctionnalités de la puce et des pins 
 mais aucun retour d'adresses 
Serial.print(TinyIRReceiverData.Address, HEX); --------> rien 
ni de commande 
Serial.print(TinyIRReceiverData.Command, HEX);--------> rien 

 rien ne s'opère depuis la bibliothèque sur attiny1616 

alors que le même programme fonctionne parfaitement sur NANO et sur MH-tiny88
merci

**Comment by LiteBrite82** [comment]
Hi @gparoz , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
